docarray/index/backends/hnswlib.py

```diff
@@ -392,6 +392,17 @@ def _get_items(self, doc_ids: Sequence[str], out: bool = True) -> Sequence[TSche
             raise KeyError(f'No document with id {doc_ids} found')
         return out_docs
 
+    def __contains__(self, item: BaseDoc):
+        if safe_issubclass(type(item), BaseDoc):
+            hash_id = self._to_hashed_id(item.id)
+            self._sqlite_cursor.execute(
+                f"SELECT data FROM docs WHERE doc_id = '{hash_id}'"
+            )
+            rows = self._sqlite_cursor.fetchall()
+            return len(rows) > 0
+        else:
+            raise NotImplementedError
```

Comment:
this does not seem a proper exception

Reply:
This PR is still in progress, I'll change it later :) It should output proper hint to users